test/v8-to-istanbul.js

```diff
@@ -28,6 +31,33 @@ describe('V8ToIstanbul', () => {
       v8ToIstanbul.source.lines.length.should.equal(48)
       v8ToIstanbul.wrapperLength.should.equal(0) // ESM header.
     })
+
+    it('handles source maps with sourceRoot', async () => {
+      const sourceFileName = 'sourcemap-source.js'
+      const sourceRoot = path.dirname(require.resolve(`./fixtures/scripts/${sourceFileName}`))
+      const absoluteSourceFilePath = path.join(sourceRoot, sourceFileName)
+      const map = new sourcemap.SourceMapGenerator({
+        file: sourceFileName,
+        sourceRoot
+      })
+      map.addMapping({
+        source: sourceFileName,
+        original: { line: 1, column: 1 },
+        generated: { line: 1, column: 1 }
+      })
```

Comment:
I'd be tempted to just pre-generate this data, and write it to a fixture ... but, at the same time, I like that this is a useful demonstration of how to auto-generate this information.

Not a blocker for me.